bin/test/setup_local_config.sh

```diff
@@ -2,12 +2,12 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 BIN_DIR=${BIN_DIR:-$(cd "${0%/*}"&&pwd)}
 cd $BIN_DIR
 
-source ./../util/util.sh
+source $BIN_DIR/../util/util.sh
```

Comment:
does it make a difference?

because I am doing `cd $BIN_DIR` above?

Reply:
it does not complain that the files not exist when running for the first time